tests/spark/test_harness.py

```diff
@@ -467,6 +478,32 @@ def test_increment_counter(self):
 
         self.assertEqual(harness_counters, reference_counters)
 
+    def test_increment_counter_non_uri_path(self):
+        input_bytes = b'one fish\ntwo fish\nred fish\nblue fish\n'
+        reference_job = self._reference_job(
+            MRCountingJob, input_bytes=input_bytes)
+
+        with reference_job.make_runner() as runner:
+            runner.run()
+            reference_counters = runner.counters()
+
+        with self.create_temp_counter_dir() as output_counter_dir:
+            harness_job = self._harness_job(
+                MRCountingJob, input_bytes=input_bytes,
+                counter_output_dir='{}'.format(output_counter_dir)
+            )
+            with harness_job.make_runner() as runner:
+                runner.run()
+                print(listdir(output_counter_dir))
+                path = join(output_counter_dir, "part-00000")
+                with open(path, 'r') as rb:
+                    counters = rb.read()
+                print('prince counters', counters)
```

Comment:
Are you leaving these `print()` calls here on purpose?

Reply:
Nopes. Thanks for catching this.